### PR TITLE
Update Microsoft.AspNetCore.AspNetCoreModule to use the correct license URL and copyright

### PR DESCRIPTION
**Pull request by natemcmaster**
Part of https://github.com/aspnet/Universe/issues/873

### CODE REVIEW

**Comment by shirhatti** [comment]
Uhh. I'm not sure about this

@ChrisSfanos @eilon Is this okay to change?

**Comment by natemcmaster** [comment]
For reference, this matches the rest of the aspnetcore packages. https://github.com/aspnet/BuildTools/blob/dev/src/Internal.AspNetCore.Sdk/build/Common.props#L13-L23

File: nuget/AspNetCore.nuspec [hunk 1]
```diff
@@ -6,10 +6,10 @@
     <version>$VERSION$</version>
     <authors>Microsoft</authors>
     <owners>Microsoft</owners>
-    <licenseUrl>http://www.microsoft.com/web/webpi/eula/net_library_eula_ENU.htm</licenseUrl>
```
Comment (natemcmaster) [inline nuget/AspNetCore.nuspec]: It's not clear to me whether apache or MIT is correct, but either way, this URL is wrong.

Comment (jkotalik) [reply nuget/AspNetCore.nuspec]: ANCM and IISIntegration used different licenses (ANCM == MIT, IISIntegration == Apache). I added both to the LICENSE.txt file after discussing with @Eilon . We may want a better solution than this.

Comment (Eilon) [reply nuget/AspNetCore.nuspec]: This package is deliberately different from other packages in ASP.NET Core, so we shouldn't change what's already there (and vetted). However, we *must* change the URLs to all be HTTPS.

Comment (natemcmaster) [reply nuget/AspNetCore.nuspec]: ok, reverted to original URL (plus https)

**Comment by natemcmaster** [comment]
:up: :date:

**Pull request merged**